src/utils/selectors.ts

```diff
@@ -71,51 +71,6 @@ export const worldUsersByIdSelector: SparkleSelector<
   Record<string, User> | undefined
 > = (state) => state.firestore.data.worldUsers;
 
-/**
- * Selector to retrieve venues from the Redux Firestore.
- *
- * @param state the Redux store
- *
- * @deprecated This selector requires all of the venues data in firebase to be loaded into memory. Find a different way.
- * @debt Refactor all places that rely on this, then remove it from the codebase
- */
-export const venuesSelector: SparkleSelector<Record<string, AnyVenue>> = (
-  state
-) => state.firestore.data.venues || {};
-
-/**
- * @deprecated This selector requires all of the venues data in firebase to be loaded into memory. Find a different way.
- * @debt Refactor all places that rely on this, then remove it from the codebase
- */
-export const orderedVenuesSelector: SparkleSelector<
-  WithId<AnyVenue>[] | undefined
-> = (state) => state.firestore.ordered.venues;
-
-/**
- * Makes a venueSelector selector for a given venueId, which when called
- * will retrieve the specified venue from the Redux Firestore.
- *
- * @param venueId the venueId to be retrieved
- * @return (state: RootState) => WithId<AnyVenue> | undefined
- *
- * @example
- *   const venueId = 'abc123'
- *   const venueSelector = useCallback(makeVenueSelector(venueId), [venueId])
- *   const venue = useSelector(venueSelector, shallowEqual)
- *
- * @deprecated This function relies on a selector that requires all of the venues data in firebase to be loaded into memory. Find a different way.
- * @debt Refactor all places that rely on this, then remove it from the codebase
- */
-export const makeVenueSelector = (venueId: string) => (
-  state: RootState
-): WithId<AnyVenue> | undefined => {
-  const venues = venuesSelector(state);
-
-  if (!venues.hasOwnProperty(venueId)) return undefined;
-
-  return { ...venues[venueId], id: venueId };
-};
-
```

Comment:
🎉 💥 👋🏻

Reply:
Make sure to also remove these aspects from the core firestore types + any hooks/other places might have referred to them as well

Reply:
As of 35d7c118100f97df09a60af2d491344e9e364a93 selectors can only be found in 2 comments inside `Playa.tsx`
```
import {
  currentVenueSelectorData,
  // orderedVenuesSelector,
} from "utils/selectors";
```
```
const Playa = () => {
  // @debt This will currently load all venues in firebase into memory.. not very efficient
  // useFirestoreConnect("venues");
  // const venues = useSelector(orderedVenuesSelector);
```